parse_file.js

```diff
@@ -0,0 +1,12 @@
+const fs = require("fs");
+
+let ResultArray;
+fs.readFile("./Assignment.txt", "utf8", function (err, contents) {
+  if(err) throw new Error(err)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/AbhishekSharma74542/Ozivatask/pullRequest?prid=8835547)
```suggestion
  if(err) {throw new Error(err)}
```